app/Http/Controllers/MessagesController.php

```diff
@@ -2,20 +2,37 @@
 
 namespace Gladiator\Http\Controllers;
 
+use Gladiator\Models\Contest;
 use Gladiator\Models\Message;
 use Gladiator\Events\QueueMessageRequest;
 
 class MessagesController extends Controller
 {
     /**
+     * Display the specified resource.
+     *
+     * @param  string  $id  Contest ID
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $messages = Message::where('contest_id', '=', $id)->get();
+
+        return view('messages.show', compact('messages'));
+    }
+
+    /**
      * Fire an event that queues a message to be sent.
      *
      * @param int $id
      */
-    public static function sendMessage($id)
+    public static function sendMessage(Message $message)
```

Comment:
@weerd Laravel 5.2 does [automatic route model binding](https://laravel.com/docs/5.2/routing#route-model-binding) so you don't have to manually bind with `Route::model`, which is really cool. _...but_ for some reason it only injects the pluralized form of the resource when automatically binding `Route::resource` routes (so like it'd only work if you injected `Message $messages` into the other REST routes).

Luckily you can [opt-in to using non-plural variable names](https://github.com/DoSomething/nightwing/blob/1149a428b2e661ea3b98bdb3681f1e56005960f8/app/Providers/RouteServiceProvider.php#L27) in your provider but it's still weeeeird.